### ISSUE TITLE
WCAG2AA.4.1_4_3.G18 - Dropdown Disabled has insufficient contrast

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#775]
**Describe the bug**
Dropdown Disabled has insufficient contrast

![dropdown](https://user-images.githubusercontent.com/9842927/68385805-7cbe1700-0152-11ea-8384-91bea79d2f2f.PNG)

**To Reproduce**
components samples:
- https://pentaho.github.io/hv-uikit-react/?path=/story/components--dropdown
- 9. Disabled multi-select dropdown
- 10. Disabled dropdown

**run**
`node C:\Git\hv-uikit-react\automation\robot\storybook\_resources/accessibility.js --url=http://localhost:9002/iframe.html?id=coredropdown--dropdown9 --standard=WCAG2AA --element="#root>div.Component-content-4" --actions="click element #header"`

**Screenshots**
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coredropdown--dropdown9',
  issues: [{
    code: 'WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail',
    type: 'error',
    typeCode: 1,
    message: 'This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.5:1. Recommendation:  change text colour to #6d6d6d.',
    context: '<p class="Typography-normalText-169 Typography-baseFontFamily-181 Typography-margin-182 Dropdown-selection-152 Dropdown-truncate-157 Dropdown-selectionDisabled-153" theme="[object Object]">Selected 3 of 12</p>',
    selector: '#header > p',
    runner: 'htmlcs',
    runnerExtras: {}
  }]
}
```
